### PR TITLE
refactor(cli): policy update api args

### PR DESCRIPTION
**Pull request by hazedav**
Cleaner approach to handling PolicyID for update

ALLY-570

### CODE REVIEW

File: api/policy.go [hunk 1]
```diff
@@ -141,14 +141,17 @@ func (svc *PolicyService) Get(policyID string) (
...
 	response PolicyResponse,
 	err error,
 ) {
-	if policyID == "" {
+	if up.PolicyID == "" {
 		err = errors.New("policy ID must be provided")
 		return
 	}
+	var policyID = up.PolicyID
+	up.PolicyID = "" // omit this for PATCH
```
Comment (afiune) [inline api/policy.go:153]: why do you need to omit the ID for PATCH?

Comment (afiune) [reply api/policy.go:153]: oh, because the ID is in the URL, gotcha... 🤔

Comment (hazedav) [reply api/policy.go:153]: That's correct.

**Comment by hazedav** [comment]
@afiune / @dmurray-lacework any additional concerns here?

**Review by afiune — APPROVED** [review]
Not from my side! 

tenor-46308957

**Pull request merged**